### ISSUE TITLE
New Line Character Doesn't Work

### ISSUE THREAD

**Issue by djamesaero** [pcolby/post-to-slack#1]
I have the below code in my GitHub Action. When it posts to slack, it does not create a new line where I have the \n in my string. Instead of posting a new line it looks like this: `GitHub Actions deployed successfully to the ${{ env.env }} environment! \n* I am a work in progress. please be patient.`

```      - name: Post to slack
        uses: pcolby/post-to-slack@v1
        with:
          url: ${{ secrets.SLACK_WEBHOOK_URL }}
          text: >-
            GitHub Actions deployed successfully to the ${{ env.env }} environment! \n
            * I am a work in progress. please be patient.```

**Comment by pcolby** [comment]
Thanks @djamesaero, this should be fixed in v1.1.0 which I just release, so give it a go and let me know how it goes for you 🙂

Cheers.

**Comment by djamesaero** [comment]
@pcolby This appears to be resolved. Thank you!
